test/scrape/6.5.1.json

```diff
@@ -1952,6 +1952,246 @@
+      "description": "Response body bytes",
+      "flag": "c",
+      "format": "B",
+      "value": 44
+    },
+    "VBE.reload_20210114_155148_19902.default.pipe_hdrbytes": {
+      "description": "Pipe request header bytes",
+      "flag": "c",
+      "format": "B",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.pipe_out": {
+      "description": "Piped bytes to backend",
+      "flag": "c",
+      "format": "B",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.pipe_in": {
+      "description": "Piped bytes from backend",
+      "flag": "c",
+      "format": "B",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.conn": {
+      "description": "Concurrent connections used",
+      "flag": "g",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.req": {
+      "description": "Backend requests sent",
+      "flag": "c",
+      "format": "i",
+      "value": 27
+    },
+    "VBE.reload_20210114_155148_19902.default.unhealthy": {
+      "description": "Fetches not attempted due to backend being unhealthy",
+      "flag": "c",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.busy": {
+      "description": "Fetches not attempted due to backend being busy",
+      "flag": "c",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.fail": {
+      "description": "Connections failed",
+      "flag": "c",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.fail_eacces": {
+      "description": "Connections failed with EACCES or EPERM",
+      "flag": "c",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.fail_eaddrnotavail": {
+      "description": "Connections failed with EADDRNOTAVAIL",
+      "flag": "c",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.fail_econnrefused": {
+      "description": "Connections failed with ECONNREFUSED",
+      "flag": "c",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.fail_enetunreach": {
+      "description": "Connections failed with ENETUNREACH",
+      "flag": "c",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.fail_etimedout": {
+      "description": "Connections failed ETIMEDOUT",
+      "flag": "c",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.fail_other": {
+      "description": "Connections failed for other reason",
+      "flag": "c",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_155148_19902.default.helddown": {
+      "description": "Connection opens not attempted",
+      "flag": "c",
+      "format": "i",
+      "value": 0
+    },
+    "VBE.reload_20210114_160902_21476.default.happy": {
```

Comment:
I just duplicated the VBE metrics with 2 different timestamps to simulate 2 VCL reloads.
Tests fail without the fix.